src/components/Application/ApplicationSummary.tsx

```diff
@@ -79,4 +73,19 @@ const ApplicationSummary: React.FC<ApplicationSummaryProps> = ({
   )
 }
 
+function getQuestion(question: ElementState, response: ResponseFull | null) {
+  return question.category === TemplateElementCategory.Question ? (
+    <Form.Field>
+      <Header as="h3" content={question.title} />
+      <Input disabled error={!response}>
+        {response?.text}
+      </Input>
+    </Form.Field>
+  ) : question.elementTypePluginCode !== 'pageBreak' ? (
+    <Form.Field>
+      <Header as="h5" content={question.title} />
```

Comment:
was wondering what this type of element would be ?

Reply:
Not sure if I understood your question. Are you asking about the `elementTypePluginCode`?

Reply:
The template element that's not a question or page break is just displayed as h5 header, was wondering if it needs to be there

Reply:
Oh, that was just to replace what will be the `SumaryView` of other elements types in the future. While its not implemented in the plugins its showing like that.